chainer/functions/array/hstack.py

```diff
@@ -53,11 +53,31 @@ def hstack(xs):
     """Concatenate variables horizontally (column wise).
 
     Args:
-        xs (list of chainer.Variable): Variables to be concatenated.
+        xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variables to be concatenated. The variables must have the
+            same shape, except in the horizontal (column wise) dimension.
```

Comment:
I tried this case and get the result below.
```
>>> x1 = np.arange(0, 12).reshape(3, 4)
>>> x2 = np.arange(0).reshape(3, 0)
>>> x2.shape
(3, 0)
>>> y = F.hstack([x1, x2])
>>> y.shape
(3, 4)
>>> y.data
array([[ 0,  1,  2,  3],
       [ 4,  5,  6,  7],
       [ 8,  9, 10, 11]])
```

I think that the result is intuitive and I think it is not necessary to add the additional description.
But I do not have enough knowledge about chainer, so please advise what to write.